extensions/shared/blocks/cover/utils.js

```diff
@@ -53,7 +53,10 @@ export function isVideoFile( file ) {
 export function isUpgradable( name ) {
 	const { unavailableReason } = getJetpackExtensionAvailability( 'videopress' );
 
-	return name && name === 'core/cover' && // upgrade only for cover block
+	return (
+		name &&
+		name === 'core/cover' && // upgrade only for cover block
```

Comment:
should we add `isCurrentUserConnected()` here?

Reply:
Since we're only interested in simple sites I think we can omit that check